content/en/users/check-in/vos/_index.md

```diff
@@ -49,7 +49,7 @@ about VO activities for EGI and for VO members (to both people and sites).
 
 A step-by-step guide for the VO registration process is provided in the
 following link:
-<https://wiki.egi.eu/wiki/PROC14_VO_Registration#VO_registration>
+<https://confluence.egi.eu/display/EGIPP/PROC14+VO+Registration>
```

Comment:
Here a proper markdown link would be better `[PROC14 VO Registration](https://confluence.egi.eu/display/EGIPP/PROC14+VO+Registration)`